Form/PhpcrOdmTypeGuesser.php

```diff
@@ -49,10 +50,16 @@ class PhpcrOdmTypeGuesser implements FormTypeGuesserInterface
 
     private $cache = array();
 
+    /**
+     * Work with 2.3-2.7 and 3.0 at the same time. drop once we switch to symfony 3.0.
+     */
+    private $entryTypeOption = 'type';
+
     public function __construct(ManagerRegistry $registry, $typeGuess = array())
     {
         $this->registry = $registry;
         $this->typeGuess = $typeGuess;
+        $this->entryTypeOption = Kernel::VERSION_ID < 20800 ? 'type' : 'entry_type';
```

Comment:
thanks. once i realized its AbstractType (and not AbstactType) i got this to work. lets see what travis makes of this.